charts/growi/README.md

```diff
@@ -0,0 +1,41 @@
+# GROWI Helm Chart
+
+## TL;DR;
+```bash
+$ helm repo add weseek https://weseek.github.io/helm-charts/
+$ helm install growi-example weseek/growi
+```
+
+## Prerequisites
+* Helm 3.0+
+   * Helm 2 is not tested
+* Kubernetes 1.13+
+
+## Parameters
+
+| Parameter        | Description                                                                                                                                                                                                                 | Default                                                                                              |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `replicaCount`     | Number of GROWI pods to run.                                                                                                                                                                                                | `1`                                                                                                  |
+| `image.repository` | GROWI Docker image name.                                                                                                                                                                                                    | `weseek/growi`                                                                                       |
+| `image.tag`        | GROWI Docker image tag.                                                                                                                                                                                                     | `3.6.2`                                                                                              |
+| `image.pullPolicy` |                                                                                                                                                                                                                             | `IfNotPresent`                                                                                       |
+| `service.type`     | Kubernetes service type.                                                                                                                                                                                                    | `NodePort`                                                                                           |
```

Comment:
Please change the default value to "ClusterIP".
After installation, we want to keep GROWI as closed as possible since everyone can create the first admin user,

Reply:
```suggestion
| `service.type`     | Kubernetes service type.                                                                                                                                                                                                    | `ClusterIP`                                                                                          |
```